.evergreen/config.yml

```diff
@@ -60,14 +61,50 @@ functions:
       params:
         file: src/expansion.yml
 
-  "install dependencies":
+  "install swift":
     - command: shell.exec
       params:
         working_dir: "src"
         script: |
           ${PREPARE_SHELL}
           SWIFT_VERSION=${SWIFT_VERSION} \
-            sh ${PROJECT_DIRECTORY}/.evergreen/install-dependencies.sh
+            sh ${PROJECT_DIRECTORY}/.evergreen/install-swift.sh
+
+  "format":
+  - command: shell.exec
+    type: test
+    params:
+      working_dir: "src"
+      script: |
+        ${PREPARE_SHELL}
+        SWIFT_VERSION=${SWIFT_VERSION} \
+          bash ${PROJECT_DIRECTORY}/.evergreen/install-tools.sh swiftformat
+        SWIFT_VERSION=${SWIFT_VERSION} \
+        ${PROJECT_DIRECTORY}/opt/swiftformat/.build/release/swiftformat --lint --quiet .
```

Comment:
when the lint fails, `--quiet` makes it so that it just prints that the lint failed, not where it failed / why. I actually think we want `--verbose` for swiftformat to include this info. For swiftlint, `--quiet` seems to still print the errors so that should be fine.

Reply:
good catch. updated